UPGRADING.rst

```diff
@@ -10,6 +10,38 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2143 Merge service accounts for indexer and service
+============================================================
+
+The following directions need to be followed for each deployment that needs to
+merge the commit.
+
+1.  Let someone who can administer authorization of SAM/TDR know of the desired
+    change in service account via Slack. Note performing these steps with out
+    the manual authorization can lead to deployment errors during
+    ``make deploy``.
+
+2.  Before upgrading to this commit, remove the old
+    shared_credentials value in the AWS Secrets Manager, run ::
```

Comment:
TMI. The reader does not need to know about shared_credentials?

Reply:
This section was not needed.